build.sbt

```diff
@@ -70,11 +70,11 @@ lazy val gemWarts =
   Warts.allBut(
     Wart.Any,                // false positives
     Wart.Nothing,            // false positives
-    Wart.Null,               // false positives
     Wart.Product,            // false positives
     Wart.Serializable,       // false positives
     Wart.Recursion,          // false positives
-    Wart.ImplicitConversion  // we know what we're doing
+    Wart.ImplicitConversion, // we know what we're doing
+    Wart.ImplicitParameter   // false positives
```

Comment:
There is an instance where I'm taking a typeclass instance but it's still triggering this warning about non-typeclass implicits. So I turned it off.